sdk/tests/conformance/more/functions/bufferSubDataBadArgs.html

```diff
@@ -67,8 +67,14 @@
         function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, 19, new Uint16Array([1,2,3]));});
     assertFail("data too large",
         function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, 0, new Uint16Array(data.concat([1])));});
-    assertOk("offset + data too large",
+    assertOk("offset + data.length = end",
         function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, 18, new Uint16Array([1,2,3]));});
+    assertOk("offset Infinity",
+        function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, Infinity, new Uint16Array([1,2,3]));});
+    assertOk("offset -Infinity",
+        function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, -Infinity, new Uint16Array([1,2,3]));});
+    assertOk("offset NaN",
+        function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, NaN, new Uint16Array([1,2,3]));});
```

Comment:
Agree with merging these into the other test, which is more complete -- these more/ tests haven't been updated in a while.

In this case, I think getBufferParameter(..., BUFFER_SIZE) will return 12 -- the original size of the array. If we want to validate the result of the call, we should add this to the WebGL 2.0 conformance suite and use getBufferSubData to ensure that (1, 2, 3) were written to the beginning of the ELEMENT_ARRAY_BUFFER.